src/pinecone_client/vector_db.py

```diff
@@ -0,0 +1,54 @@
+import pinecone
+from langchain.embeddings.openai import OpenAIEmbeddings
+from dotenv import load_dotenv
+import os
+from langchain.vectorstores import Pinecone
```

Comment:
```suggestion
from langchain.vectorstores.pinecone import Pinecone
```
おそらくこのPathにしないと動作させた時にエラーが出ると思いますが、いかがでしょうか？